wpuf.php

```diff
@@ -412,6 +431,28 @@ function wpuf_loader() {
         }
     }
 
+    /**
+     * Latest Pro Activation Message
+     *
+     * @return void
+     */
+
+    function wpuf_latest_pro_activation_notice() {            
```

Comment:
Use a message like this:

![image](https://user-images.githubusercontent.com/153669/50877352-ac4da700-13fb-11e9-8ade-8d96ba7296f4.png)